services/parsePatch.service.js

```diff
@@ -1,21 +1,25 @@
 "use strict";
-const mongoose = require("mongoose");
 
 module.exports = {
-    parsePatch: function(req, model, done) {
-        setMatchingAttributes(model).then(() => {
-            done();
-        });
+    /**
```

Comment:
I would consider moving this function to the middlewares folder and calling it parsePatch.middleware.js, because it would be used as a middleware.